### PR TITLE
WIP: 12.0 mig account payment batch process

### PR DESCRIPTION
**Pull request by ovnicraft**

### CODE REVIEW

File: account_payment_batch_process/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Open Source Integrators
+# <https://www.opensourceintegrators.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Batch Payments Processing",
+    "summary": "Process Payments in Batch",
+    "version": "11.0.1.0.0",
```
Comment (kittiu) [inline account_payment_batch_process/__manifest__.py]: Please change version to 12.0.1.0.0

**Review by kittiu — COMMENTED** [review]
Looks like this module is not ready for review?
Will come back later.

File: account_payment_batch_process/__manifest__.py (hunk 1 above)
Comment (dreispt) [inline account_payment_batch_process/__manifest__.py]:
```suggestion
    "version": "12.0.1.0.0",
```

**Comment by astirpe** [comment]
Similar PR here: https://github.com/OCA/account-payment/pull/337

**Comment by ovnicraft** [comment]
i close this in favor #337

**Pull request closed without merging**